http/src/main/resources/init-resources/init-actions.sh

```diff
@@ -158,6 +145,19 @@ ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
 
 # Only initialize tool and proxy docker containers on the master
 if [[ "${ROLE}" == 'Master' ]]; then
```

Comment:
For a future PR, it would be sweet to break the roles out into separate files

Reply:
It is a bit tricky due to how dataproc initialization works, but might be a good investigation for the terra-docker refactoring work we want to do :)